src/main/scala/viper/server/core/ViperCache.scala

```diff
@@ -351,7 +352,8 @@ object ViperCacheHelper {
     def addIdxToHash(hash: String): String = idx.toString + hash
 
     node match {
-      case m: Method => addIdxToHash(removeBody(m).entityHash)
+      case m: Method => removeBody(m).entityHash
```

Comment:
add a comment that the index is  not added to members (incl. methods) because their ordering in the AST does not matter as they have to be uniquely identifiable anyway (e.g. via their name)